websites/D/diziwatch/presence.ts

```diff
@@ -0,0 +1,81 @@
+const presence = new Presence({
+		clientId: "1214974615202299994",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://i.imgur.com/45R11f9.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: Assets.Logo,
+	};
+	switch (document.URL) {
```

Comment:
Prefer `document.location.href` over `document.URL`. 
However, for the use case here, `document.location.pathname` may be a better choice for maintainability.